src/components/HeaderWithBackButton/index.js

```diff
@@ -51,6 +51,7 @@ function HeaderWithBackButton({
         horizontal: 0,
     },
     threeDotsMenuItems = [],
+    children,
```

Comment:
We should change this to `children = null`.

Reply:
Updated!